wiki/Contests/osu!trivium_Quiz/2020/en.md

```diff
@@ -0,0 +1,80 @@
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 5 | ![][flag_AU] [Beat43210](https://osu.ppy.sh/users/5664171), ![][flag_MY] [cdh](https://osu.ppy.sh/users/2403621), ![][flag_US] [icedtea](https://osu.ppy.sh/users/9631242) |
+
+## Quiz answers
+
+Below is a list of links to answers to each question. Clicking on any of them will lead to the respective timestamp in the results announcement VOD where the answer is explained.
+
+| Difficulty | Answers |
+| :-: | :-- |
+| Easy | [1](https://youtu.be/KIgdjSleasg?t=510), [2](https://youtu.be/KIgdjSleasg?t=724), [3](https://youtu.be/KIgdjSleasg?t=816), [4](https://youtu.be/KIgdjSleasg?t=1092), [5](https://youtu.be/KIgdjSleasg?t=1270), [6](https://youtu.be/KIgdjSleasg?t=1379), [7](https://youtu.be/KIgdjSleasg?t=1457), [8](https://youtu.be/KIgdjSleasg?t=1550), [9](https://youtu.be/KIgdjSleasg?t=1664), [10](https://youtu.be/KIgdjSleasg?t=1726) |
+| Normal | [11](https://youtu.be/KIgdjSleasg?t=1851), [12](https://youtu.be/KIgdjSleasg?t=1993), [13](https://youtu.be/KIgdjSleasg?t=2080), [14](https://youtu.be/KIgdjSleasg?t=2189), [15](https://youtu.be/KIgdjSleasg?t=2310), [16](https://youtu.be/KIgdjSleasg?t=2413), [17](https://youtu.be/KIgdjSleasg?t=2524), [18](https://youtu.be/KIgdjSleasg?t=2634), [19](https://youtu.be/KIgdjSleasg?t=2727), [20](https://youtu.be/KIgdjSleasg?t=2886) |
+| Hard | [21](https://youtu.be/KIgdjSleasg?t=3276), [22](https://youtu.be/KIgdjSleasg?t=3389), [23](https://youtu.be/KIgdjSleasg?t=3515), [24](https://youtu.be/KIgdjSleasg?t=3613), [25](https://youtu.be/KIgdjSleasg?t=3739), [26](https://youtu.be/KIgdjSleasg?t=3847), [27](https://youtu.be/KIgdjSleasg?t=3970), [28](https://youtu.be/KIgdjSleasg?t=4068), [29](https://youtu.be/KIgdjSleasg?t=4102), [30](https://youtu.be/KIgdjSleasg?t=4150) |
+| Insane | [31](https://youtu.be/KIgdjSleasg?t=4422), [32](https://youtu.be/KIgdjSleasg?t=4600), [33](https://youtu.be/KIgdjSleasg?t=4710), [34](https://youtu.be/KIgdjSleasg?t=4858), [35](https://youtu.be/KIgdjSleasg?t=4983), [36](https://youtu.be/KIgdjSleasg?t=5149), [37](https://youtu.be/KIgdjSleasg?t=5238), [38](https://youtu.be/KIgdjSleasg?t=5626), [39](https://youtu.be/KIgdjSleasg?t=5720), [40](https://youtu.be/KIgdjSleasg?t=5872) |
+| Extra | [41](https://youtu.be/KIgdjSleasg?t=6080), [42](https://youtu.be/KIgdjSleasg?t=6202), [43](https://youtu.be/KIgdjSleasg?t=6347), [44](https://youtu.be/KIgdjSleasg?t=6495), [45](https://youtu.be/KIgdjSleasg?t=6762), [46](https://youtu.be/KIgdjSleasg?t=6928), [47](https://youtu.be/KIgdjSleasg?t=7040), [48](https://youtu.be/KIgdjSleasg?t=7172), [49](https://youtu.be/KIgdjSleasg?t=7400), [50](https://youtu.be/KIgdjSleasg?t=7687) |
+
+## Ruleset
+
+- You cannot ask [mangomizer](https://osu.ppy.sh/users/1893718) for information, hints, or answers.
```

Comment:
```suggestion
- You cannot ask ![][flag_GB] [mangomizer](https://osu.ppy.sh/users/1893718) for information, hints, or answers.
```

Reply:
I don't think the flag is necessary here (same goes for other ruleset suggestions of yours)